src/models/User.ts

```diff
@@ -0,0 +1,39 @@
+import { DataTypes, Model } from "sequelize";
+import sequelize from "../database/db";
+
+export enum UserType {
+  DE = "DE", // Public Administration delegate
```

Comment:
storage is quite cheap, why don't we use extended names? `Delegate`, `Manager`, etc...